resource-managers/kubernetes/integration-tests/dev/dev-run-integration-tests.sh

```diff
@@ -189,7 +189,6 @@ properties+=(
   -Dspark.kubernetes.test.jvmImage=$JVM_IMAGE_NAME
   -Dspark.kubernetes.test.pythonImage=$PYTHON_IMAGE_NAME
   -Dspark.kubernetes.test.rImage=$R_IMAGE_NAME
-  -Dlog4j.logger.org.apache.spark=DEBUG
```

Comment:
This seems not work with log4j 2.x. So for K8S decommission integration tests which check logs, I updated the test code to set up a log4j2.properties.